apps/condo/domains/notification/tasks.js

```diff
@@ -100,6 +101,53 @@ async function deliverMessage (messageId) {
         sender: message.sender,
     }
 
+    if (message.organization) {
+        const messageOrganizationBlackListWhere = {
```

Comment:
looks like this logic can be moved to helper

Reply:
moved to `checkMessageTypeInBlackList` helper